.travis.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       script: ./gradlew clean build
 
     - stage: snapshot
-      if: (branch = master OR branch =~ /^r[0-9]+\.[0-9]+/) AND NOT (type = pull_request)
+      if: (branch = master OR branch =~ /^r[0-9]\.[0-9][\S]*/) AND NOT (type = pull_request)
```

Comment:
If we end up deciding to use our branch name as `r0.8-spark3.0` which is the same as the Flink connector. We need to change the regular expression to match the `-spark3.0` substring. The current regular expression only matches the `r0.8`. So I followed the [Travis set up](https://github.com/pravega/flink-connectors/blob/763ecfcda288ce57e492e7c9015d4c536d0c95a0/.travis.yml#L39) in the Flink connector to make the changes. 

Taking `r0.8-spark3.0` as an example:

- `^r[0-9]+\.[0-9]+` will only match  `r0.8`
- `^r[0-9]\.[0-9][\S]*` will match the full string `r0.8-spark3.0`

Reply:
By taking out the first `+`, it will no longer match something like `r10.0` because it will only accept a single digit before the period. This looks like a problem here and in the Flink connector.
The pattern should be `^r[0-9]+\.[0-9][\S]*`
Also, add a comment in the .travis.yml file like this:
```
# Publish the artifact if the branch is `master` or is something like `r0.8-spark3.0` or `r0.9`.
```